lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2301,6 +2302,10 @@ decodeTransaction ctx (ApiT wid) (ApiSerialisedTransaction (ApiT sealed)) = do
         , assetsMinted = ApiT toMint
         , assetsBurned = ApiT toBurn
         , certificates = map (toApiAnyCert acct acctPath) allCerts
+        , deposits =
+                map (const (Quantity . fromIntegral . unCoin . W.stakeKeyDeposit $ pp)) $
```

Comment:
An interesting, but presumably unavoidable thing is that the deposit value could in theory be changed to 3 ada next epoch, and then when you call decodeTx on your old, first delegation tx, it shows as having 3 ada in deposit even though you just paid 2 ada.

I think this is the case at least.